packages/admin/src/pages/dashboard/index.page.tsx

```diff
@@ -54,6 +66,7 @@ const Dashboard = ({ schemes, userDetails }: DashboardProps) => {
 interface DashboardProps {
   schemes: Scheme[];
   userDetails: UserDetails;
+  oneLoginTransferErrorEnabled: boolean;
 }
```

Comment:
Overall looks perfecto, would just refactor the PageProps types we've updated like this & use `InferProps<typeof getServerSideProps>` instead!